chart/kubeapps/templates/NOTES.txt

```diff
@@ -8,6 +8,16 @@ Kubeapps can be accessed via port {{ .Values.frontend.service.port }} on the fol
 
    {{ template "kubeapps.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
 
+{{- $reposWithOrphanSecrets := include "kubeapps.repos-with-orphan-secrets" . }}
+{{- if ne $reposWithOrphanSecrets "" }}
+
+CAVEAT:
+  Some App Repositories has been installed with a custom CA or authorization header.
+This generates secrets that won't be cleaned up if the repository is deleted through the Web application.
```

Comment:
can we indent these two lines too? It looks a little weird to me:

![screen shot 2019-02-06 at 11 34 09](https://user-images.githubusercontent.com/1544861/52368398-270de000-2a03-11e9-816b-b2aef516ba20.png)